src/components/header/header.component.tsx

```diff
@@ -13,12 +13,13 @@ import MingcuteLightModeFill from "@/icons/MingcuteLightMode";
 export default function HeaderComponent() {
   const pathname = usePathname();
 
-  const [lightMode, setLightMode] = useState(() => {
-    if (typeof window !== "undefined") {
-      return localStorage.getItem("lightMode") === "true";
-    }
-    return false;
-  });
+ const [lightMode, setLightMode] = useState(() => {
+   if (typeof window !== "undefined") {
+     const savedTheme = localStorage.getItem("lightMode");
+     return savedTheme ? savedTheme === "true" : true;
+   }
+   return true; 
```

Comment:
بهتره برعکس چک کنید
اونی که کوتاه‌تره اول بیاد که اونی که بلندتره زیادی Indent نخوره

```suggestion
   if (typeof window === "undefined") {
     return true;
   }
   
   const savedTheme = localStorage.getItem("lightMode");
   return savedTheme ? savedTheme === "true" : true;
```